packages/next/src/client/components/router-reducer/fetch-server-response.ts

```diff
@@ -30,6 +30,12 @@ import { callServer } from '../../app-call-server'
 import { PrefetchKind } from './router-reducer-types'
 import { hexHash } from '../../../shared/lib/hash'
 
+export interface FetchServerResponseOptions {
```

Comment:
curious what the rationale is for including these properties in `options`, but not `flightRouterState`?

Reply:
Good question, which I debated as well. In the end, the jsdoc comment was kinda guiding me:
>Fetch the flight data for the provided url. Takes in the current router state to decide what to render server-side.
So it felt like those two params are primary and secondary, whereas the rest could be considered options.

Reply:
Fair enough. I think technically both `nextUrl` and `prefetchKind` _also_ determine what to render server side, but I don't have a strong opinion either way 👍

Reply:
I ended up moving `flightRouterState` also into the options.